src/module/types.d.ts

```diff
@@ -66,6 +68,31 @@ declare interface SWNRCharacterData extends SWNRLiving, SWNREncumbrance {
   stats: { [key in SWNRStats]: SWNRStat };
 }
 
+declare interface SWNRDroneData extends SWNRCombatantData {
+  model:
+    | "primitiveDrone"
+    | "voidHawk"
+    | "stalker"
+    | "cuttlefish"
+    | "ghostwalker"
+    | "sleeper"
+    | "pax"
+    | "alecto"
+    | "other";
```

Comment:
This limits extendability. Should just be a plain string.
```suggestion
  model: string;
```

Reply:
Gotcha. Change was made.